source/OctoPack.nuspec

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-  <metadata>
+  <metadata minClientVersion="2.5">
```

Comment:
This was introduced in [NuGet 2.5 in 2013] (https://devblogs.microsoft.com/nuget/a-quick-note-about-minclientversion-feature/), I think we're safe.

Reply:
Well, I was here, I came across the information, and so I added it. No harm no foul, really. Hopefully, no one's using Visual Studio 2010 or 2012 trying to build OctoPack at this time! 😄  The more important change is the one above to be a bit more in line with "best practices".